client/v2/offchain/sign.go

```diff
@@ -63,7 +63,7 @@ func Sign(ctx client.Context, rawBytes []byte, fromName, signMode, indent, encod
 		return "", err
 	}
 
-	tx, err := sign(ctx, fromName, digest, getSignMode(signMode))
+	tx, err := sign(ctx, fromName, digest, getSignMode(flags.SignModeTextual))
```

Comment:
I think there is more simplification that can be done than just [this](https://github.com/cosmos/cosmos-sdk/pull/18626/commits/19ba61fb6ead0633a5d767c6966ef332b8167991) if we accept only textual.

Reply:
yes absolutely, the `getSignMode` and tests can be deleted. But I think its a good idea to keep as its a work already done that can be useful in the future. At least the tests about `getSignMode` not big deal.

Reply:
We do want to really limit the amount of signer in the sdk (long term goal), I'd argue that it is better to delete all the code that we won't use now instead of later.